surfsense_backend/alembic/versions/13_add_google_drive_connector_enums.py

```
@@ -0,0 +1,68 @@
+"""Add GOOGLE_DRIVE_CONNECTOR to SearchSourceConnectorType and DocumentType enums
+
+Revision ID: 13
+Revises: 12
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
```

Ruff (0.12.2)

`sqlalchemy` imported but unused

Remove unused import: `sqlalchemy`

(F401)